app/views/shared/_loss_report_modal.html.erb

```diff
@@ -0,0 +1,19 @@
+<div id="loss-report-modal" class="eui-modal-content" >
+  <a href="javascript:void(0);" class="modal-close float-r" id="close-loss-report-modal"><i class="fa fa-times"></i><span class="is-invisible">Close</span></a>
+  <div>
+    <h4>Conversion to UMM Format Required</h4>
+    <p>The native format of this collection is <%= options[:native_format].split('/').last.gsub('+xml','').upcase %>. Editing this record using the MMT will convert it to UMM-JSON, which may result in data loss.</p>
+    <p style="display: inline-flex">
```

Comment:
for some reason one or two buttons were not aligning horizontally and this `display: inline-flex` solved that issue. Let me know if there's a better way to do this.

Reply:
I think we talked about this modal and modifying some of the classes, so this style is no longer necessary, but leaving the above comment as it is something to keep in mind.

Reply:
Depending on how you solve the modal-close comment above, this could easily be added to that.  Either way, I think you should remove the inline styling.